internal/namespace/aliasing.go

```diff
@@ -0,0 +1,81 @@
+package namespace
+
+import (
+	"fmt"
+	"sort"
+)
+
+// computePermissionAliases computes a map of aliases between the various permissions in a
+// namespace. A permission is considered an alias if it *directly* refers to another permission
+// or relation without any other form of expression.
+func computePermissionAliases(typeSystem *ValidatedNamespaceTypeSystem) (map[string]string, error) {
```

Comment:
Do we expect this to save us enough time to be worth it? If there were more than one alias of the same relation or permission, they would share a cache key and only 1/n calls would actually be invoked. Am I missing something that makes this worth adding?

Reply:
In the case where there isn't any caching yet, and where there is a chain of aliasing (2-3 at least), it absolutely could save time because we'd be redispatching from node to node for each aliasing.